src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -2,191 +2,169 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.Serialization;
+using System.Xml;
+
 namespace System.ServiceModel.Syndication
 {
-    using System;
-    using System.Diagnostics;
-    using System.Globalization;
-    using System.Runtime.Serialization;
-    using System.Xml;
-    using DiagnosticUtility = System.ServiceModel.DiagnosticUtility;
-
     public delegate bool TryParseDateTimeCallback(XmlDateTimeData data, out DateTimeOffset dateTimeOffset);
     public delegate bool TryParseUriCallback(XmlUriData data, out Uri uri);
 
     [DataContract]
     public abstract class SyndicationFeedFormatter
     {
-        private SyndicationFeed _feed;
-
         protected SyndicationFeedFormatter()
         {
-            _feed = null;
+            Feed = null;
             DateTimeParser = GetDefaultDateTimeParser();
         }
 
         protected SyndicationFeedFormatter(SyndicationFeed feedToWrite)
         {
-            if (feedToWrite == null)
-            {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("feedToWrite");
-            }
-            _feed = feedToWrite;
+            Feed = feedToWrite ?? throw new ArgumentNullException(nameof(feedToWrite));
             DateTimeParser = GetDefaultDateTimeParser();
         }
 
-        public SyndicationFeed Feed
-        {
-            get
-            {
-                return _feed;
-            }
-        }
+        public SyndicationFeed Feed { get; private set; }
 
         public TryParseUriCallback UriParser { get; set; } = DefaultUriParser;
 
         // Different DateTimeParsers are needed for Atom and Rss so can't set inline
         public TryParseDateTimeCallback DateTimeParser { get; set; }
 
-        internal virtual TryParseDateTimeCallback GetDefaultDateTimeParser()
-        {
-            return NotImplementedDateTimeParser;
-        }
+        internal virtual TryParseDateTimeCallback GetDefaultDateTimeParser() => NotImplementedDateTimeParser;
 
         private bool NotImplementedDateTimeParser(XmlDateTimeData XmlDateTimeData, out DateTimeOffset dateTimeOffset)
         {
-            dateTimeOffset = default(DateTimeOffset);
+            dateTimeOffset = default;
             return false;
         }
 
-        public abstract string Version
-        { get; }
+        public abstract string Version { get; }
 
         public abstract bool CanRead(XmlReader reader);
 
         public abstract void ReadFrom(XmlReader reader);
 
         public override string ToString()
         {
-            return String.Format(CultureInfo.CurrentCulture, "{0}, SyndicationVersion={1}", this.GetType(), this.Version);
+            return string.Format(CultureInfo.CurrentCulture, "{0}, SyndicationVersion={1}", GetType(), Version);
         }
```

Comment:
Need consistent conversion to delegate-bodied methods. This method can ben made delegate-bodied and wasn't, but you've converted some other methods.